homeassistant/components/fronius/sensor.py

```diff
@@ -224,7 +233,7 @@ async def _update(self) -> dict:
         """Return values of interest."""
 
     @callback
-    def register(self, sensor):
+    def register(self, sensor: SensorEntity):
```

Comment:
If typing, please fully type a method, in this case, the return type is missing.